flocker/control/_diffing.py

```diff
@@ -15,6 +17,7 @@
     pvector,
     pvector_field,
 )
+from pyrsistent._transformations import _get
```

Comment:
This looks private.  If so, file an issue with upstream to expose this functionality publicly, link to it from here?